### ISSUE TITLE
DWS ascent facet filter not working

### ISSUE THREAD

**Issue by scd** [theCrag/website#3826]
**What happened?**

Deep Water Solo tick type not showing any results in the ascents facet.

https://www.thecrag.com/ascents/with-ascent-gear-style/dws

There should be some ascents in the facet above.

**Comment by scd** [comment]
Note that this is a bug. We should not be using the dws abbreviation in the url search string, or we should interpret dws in the server. Cannonical

https://www.thecrag.com/ascents/with-ascent-gear-style/deep-water-solo

**Comment by scd** [comment]
Ok I have finally worked out what this is. The `with-ascent-gear-style` is really misnamed from when we introduced cpr and refers to the cpr gear style. 

Now that we have properly introduced the climbed gear style for the ascent it is very confusing. From next release this tag will now be called `with-climbed-gear-style` and will pick up DWS from the new tick flow.

We will probs get complaints about the old tick flow not being picked up by this, but we need to sit on this for a bit to work out what to do.
